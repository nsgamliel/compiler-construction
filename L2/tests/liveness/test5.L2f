(:myF
0

  %variableVIZ0XN784J <- rbx

  rbx <- 5
  %myVar1 <- rbx

  rbx <- %variableVIZ0XN784J
  return
)
