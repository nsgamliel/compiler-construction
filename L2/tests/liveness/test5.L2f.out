(
(in
(r12 r13 r14 r15 rax rbp rbx)
(r12 r13 r14 r15 rax rbp %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp rbx %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp rbx)
)

(out
(r12 r13 r14 r15 rax rbp %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp rbx %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp %variableVIZ0XN784J)
(r12 r13 r14 r15 rax rbp rbx)
()
)

)
